jetstream/analysis.py

```diff
@@ -27,14 +31,13 @@ class Analysis:
 
     project: str
     dataset: str
-    config: AnalysisConfiguration
```

Comment:
When dask distributes tasks across multiple processes it needs to pickle `AnalysisConfiguration`. However, pickling `AnalysisConfiguration` results in a bunch of errors. For example, there are issues with pickling and unpickling the `Statistics`. So, in order to avoid these issues AnalysisConfiguration needs to be removed as an attribute and we only pass in the parameters we actually need into `dask.delayed` functions

Reply:
Pickle is so bad.

I think it ought to be possible to serialize an AnalysisConfiguration instead of blowing up our APIs, though. We should at least be able to write code to serialize/deserialize things from JSON and write some [serializer hooks](https://distributed.dask.org/en/latest/serialization.html#dask-serialization-family).